src/firefly/plans/regions_display.py

```diff
@@ -164,14 +164,11 @@ def update_total_time(self):
         self.ui.label_sec_total.setText(str(secs_total))
 
     def time_calculate_method(self, detector_time):
```

Comment:
Can we rename this function to better describe what it does? ``time_calculate_method`` sounds like it returns the method that will be used for calculating time. I think what it actually returns is the ``time_per_scan`` or something, right?